sqlglot/parser.py

```diff
@@ -2944,20 +2944,20 @@ def _parse_sort(self, exp_class: t.Type[E], token: TokenType) -> t.Optional[E]:
 
     def _parse_ordered(self) -> exp.Ordered:
         this = self._parse_conjunction()
-        self._match(TokenType.ASC)
 
-        is_desc = self._match(TokenType.DESC)
+        asc = self._match(TokenType.ASC)
+        desc = self._match(TokenType.DESC) or (asc and False)
```

Comment:
what and False?

Reply:
It could be rewritten as `None if asc is None else False`

Reply:
ok, because you want it to be none? that's fine

Reply:
yes that was the intent